samples/msal-core-samples/VanillaJSTestApp/app/ie11-sample/test/browser.spec.ts

```diff
@@ -0,0 +1,98 @@
+import puppeteer from "puppeteer";
```

Comment:
I think we can go without an E2E test for this because it runs on Chromium anyway

Reply:
Pardon my ignorance, just remove the the test file then? I suppose this would have applied to other VanillaJSTestApp samples too..

Reply:
Right. We don't have a mechanism to test browser compatibility at the moment so I think the tests in the other samples already cover the basic flows

Reply:
Got it -removed!